cmd/github-labels/labels.yaml.in

```diff
@@ -0,0 +1,538 @@
+
+  - name: related/k8s
+    description: Kubernetes
+    category: related
+    color: DEFAULT_COLOUR
+
+  - name: related/qemu
+    description: QEMU
+    category: related
+    color: DEFAULT_COLOUR
+
+  - name: related/runc
+    description: Runc
+    category: related
+    color: DEFAULT_COLOUR
+
+  - name: release-gating
+    description: Release must wait for this to be resolved before release
+    category: release
+    color: DEFAULT_COLOUR
+
+  - name: resource-hog
+    description: System uses too many resources (such as memory)
+    category: behaviour
+    color: DEFAULT_COLOUR
+
+  - name: resource-leak
+    description: System does not free resources (such as memory)
+    category: behaviour
+    color: DEFAULT_COLOUR
+
+  - name: rfc
+    description: Requires input from the team
+    category: question
+    color: DEFAULT_COLOUR
+
+  - name: security
+    description: Potential or actual security issue
+    category: security
+    color: ff0000
+
+  - name: size/huge
+    description: |
+      Largest and most complex task (probably needs breaking into small
+      pieces)
+    category: sizing
+    color: DEFAULT_COLOUR
+
+  - name: size/large
+    description: Task of significant size
+    category: sizing
+    color: DEFAULT_COLOUR
+
+  - name: size/medium
+    description: Average sized task
+    category: sizing
+    color: DEFAULT_COLOUR
+
+  - name: size/small
+    description: Small and simple task
+    category: sizing
+    color: DEFAULT_COLOUR
+
+  - name: size/tiny
+    description: Smallest and simplest task
+    category: sizing
+    color: DEFAULT_COLOUR
+
+  - name: stale
+    description: Issue or PR was not updated in a timely fashion
+    category: resolution
+    color: DEFAULT_COLOUR
+
+  - name: team/ci
+    description: Need Continuous Integration Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/developer
+    description: Need Developer Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/documentation
+    description: Need Documentation Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/kernel
+    description: Need Kernel Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/metrics
+    description: Need Metrics Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/packaging
+    description: Need Packaging Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: team/test
+    description: Need Test Team input
+    category: team
+    color: DEFAULT_COLOUR
+
+  - name: unreliable
+    description: Part of the system is not stable
+    category: behaviour
+    color: DEFAULT_COLOUR
+
+  - name: wont-fix
+    description: Issue will not be fixed (not a good use of limited resources)
+    category: resolution
+    color: DEFAULT_COLOUR
+
+  - name: wrong-repo
+    description: Raised in incorrect repository
+    category: resolution
+    color: DEFAULT_COLOUR
```

Comment:
I think my eyes have label blur 😱 That's a lot of categories and labels. OK, so, looking at some of the defacto-ish projects that have formalised their labels (k8s, rkt, docker), it is not a dissimilar list or size - but kudos to the first (or will that be 2nd) person who reads and digests the whole list ;-)

Reply:
Feel free to comment on particular labels. I'm sure there are issues and we can argue about individual labels or categories on the PR.

Oh, and spare a thought for the poor author who had to put this whole thing together... and yes, it all took a *lot* of my time ;)